zap/src/main/java/org/parosproxy/paros/model/Session.java

```diff
@@ -1495,6 +1496,40 @@ public Context importContext(File file)
         String name = config.getString(Context.CONTEXT_CONFIG_NAME);
         validateContextName(name);
 
```

Comment:
This should be done in `importContext(ZapXmlConfiguration)`.

Reply:
But then we would be back to passing a Boolean or consumer and how was then any better than the earlier impl with a file?

Reply:
Okay, right so it just won't matter if it happens again. Okay 👍